content/ios/user.md

```diff
@@ -79,11 +91,15 @@ if (container.currentUserRecordID) {
 
 ## Logging out
 
+Of course, you also would want to allow users to log out of their accounts by using `logoutWithCompletionHandler:`.
+
 ```obj-c
 [container logoutWithCompletionHandler:nil];
 ```
 
-## Changing user password
+## Changing password
+
+Sometimes users would want to change their passwords to secure their accounts. The following code illustrates the process.
```

Comment:
I think this can be expressed using "To change the password of the current user:"